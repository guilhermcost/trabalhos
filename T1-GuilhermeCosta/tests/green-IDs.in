aAbB
c1CdDeE
fF2gG
hHi3IjJ
kKlL
m n 
oOpPqQ
rRsStTuU
vVwWxX
yYzZ
aA0123456789zZ
b1B2

