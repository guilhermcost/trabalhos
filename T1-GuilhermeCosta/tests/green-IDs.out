(1,ID,"aAbB")
(2,ID,"c1CdDeE")
(3,ID,"fF2gG")
(4,ID,"hHi3IjJ")
(5,ID,"kKlL")
(6,ID,"m")
(6,ID,"n")
(7,ID,"oOpPqQ")
(8,ID,"rRsStTuU")
(9,ID,"vVwWxX")
(10,ID,"yYzZ")
(11,ID,"aA0123456789zZ")
(12,ID,"b1B2")
